pkg/osquery/runtime/history/history_test.go

```diff
@@ -189,3 +172,23 @@ func TestLatestInstance(t *testing.T) { // nolint:paralleltest
 		})
 	}
 }
+
+func newTestBoltDb(t *testing.T, instancesToAddToDb ...*Instance) *bbolt.DB {
+	db, err := bbolt.Open(fmt.Sprintf("%s/%s", t.TempDir(), "osquery_instance_history_test.db"), 0600, nil)
```

Comment:
`ioutil.TempFile(t.TempDir(), "osquery_instance_history_test.db")` I think

Reply:
this that will handle it, here is docs on t.TempDir()

> TempDir returns a temporary directory for the test to use. The directory is automatically removed by Cleanup when the test and all its subtests complete. Each subsequent call to t.TempDir returns a unique directory; if the directory creation fails, TempDir terminates the test by calling Fatal.

Reply:
nvm, I stand corrected, windows threw an error using t.TempDir() =(

Reply:
bbolt creates the db if it doesn't exist

Reply:
I agree the docs on `t.TempDir()` sound good. I wonder if the windows error was because you're using `Sprintf` to join with `/` instead of `filepath.Join` ?